README.md

```diff
@@ -222,6 +224,12 @@ export default JSONAPIAdapter.extend(AdapterMixin, {
 });
 ```
 
+# Migrations
+
+## Migration from v1.6.0
+If you used `buildUrl` in you adapter to customize URL, then you can use from now on [urlForCustomAction](#urlForCustomAction).
```

Comment:
Migrations guide should be included only in release notes.
It should be much much longer :D As in #27.

Reply:
Removed from README. I've added small description to PR comment. IMO rest details are covered in your notes in #27 - these notes can be combined and used later for release notes/migration guide.
If you want to some more changes/more detailed description, let me know (with points which I can describe) ;)